productPage/productPage.css

```diff
@@ -0,0 +1,242 @@
+@import url('https://fonts.googleapis.com/css2?family=Karla:wght@200;300;400;500;600;700;800&family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+/* * {
+    font-family: 'Montserrat', 'sans-serif';
+} */
+
+.grid-container {
+    display: grid;
+    grid-template-columns: (3, 1fr);
+    grid-template-areas:
+        'navbar navbar navbar'
+        'sidebar content content'
+        'sidebar content content'
+        'sidebar content content'
+}
+
+.header {
+    grid-area: navbar;
+    /* border: 2px solid red; */
+}
+
+
+/* Filter Section */
+.filter {
+    /* border: 2px solid red; */
+    grid-area: sidebar;
+    width: 29.4rem;
+    padding: 1.5rem;
+    margin: 15rem auto 5rem auto;
+    
+}
+
+.filters-cta-button {
+    margin-left: 10rem;
+}
+
+.back-btn {
+    width: 10.1rem;
+    height: 4.4rem;
+    left: 4.4rem;
+    top: 21.4rem;
+    border-radius: 1.2rem;
+    border: none;
+    background: #2780F2;
```

Comment:
you can use a var for color and mention it in the root, then you the variables instead of direct color code.

Reply:
done!